src/styles/scss/margins.scss

```diff
@@ -1,5 +1,5 @@
-$xs: 0.125rem;
-$sm: 0.25rem;
-$md: 0.375rem;
-$lg: 0.5rem;
-$xl: 0.625rem;
+$xs: 0.25rem;
+$sm: 0.5rem;
```

Comment:
Need to update values in story

![image](https://user-images.githubusercontent.com/36342248/202226127-44cdc08d-2d21-4306-bc44-d7136b3670c0.png)